backend/atlas_asset_pipelines/concat_corpus/job.py

```diff
@@ -0,0 +1,82 @@
+import gc
+import logging
+import os
+from typing import List
+
+import anndata
+import tiledb
+
+from backend.atlas_asset_pipelines.concat_corpus.load import load_h5ad
+from backend.atlas_asset_pipelines.concat_corpus.transform import apply_pre_concatenation_filters
+from backend.atlas_asset_pipelines.concat_corpus.validate import validate_dataset_properties, validate_corpus_load
+from backend.wmg.data.schemas.corpus_schema import INTEGRATED_ARRAY_NAME
+from backend.wmg.data.tiledb import create_ctx
+from backend.wmg.data.utils import get_all_dataset_ids
+
+logger = logging.getLogger(__name__)
+logging.basicConfig(level=logging.INFO)
+
+
+def build_concat_corpus(dataset_directory: List, corpus_path: str, validate: bool = False):
```

Comment:
Currently, this package is building the _integrated_ corpus, which is a step beyond a _concatenated_ corpus.  So for now, we might rename this package dir and this method (and other places?) to use `integrated` instead of `concatenated`.  From other discussions we've had, we'll likely end up building a `concatenated` corpus, and I could see that as another ETL-styled package, whose output would be the input to this ETL package